README.md

```diff
@@ -6,12 +6,14 @@ The coordination service for running drivers on [PlaceOS](https://place.technolo
 
 ## Testing
 
-- `$ ./test`
+- `$ ./test` (tear down the docker-compose environment)
+- `$ ./test --watch` (only run tests on change)
 - `$ docker-compose down` when you are done with development work for the day
```

Comment:
It would be could to capture `ctrl-c` and tear down the docker-compose network for `./test --watch`

```suggestion
```

Reply:
`docker-compose run --rm core` automatically removes `core` after exit code. So I just add another line in the sh script to `docker-compose down`. Is this all good now?

Reply:
it would be a little bit cleaner if you tore down the environment